boards/imix/src/main.rs

```diff
@@ -446,6 +461,7 @@ pub unsafe fn reset_handler() {
     rf233.start();
 
     imix.pconsole.start();
+    imix.permissions.start();
```

Comment:
You don't _have_ to have a `start` function, and it isn't currently doing anything, so it's fine to just exclude it. Some other capsules have `start` functions because they actually need to initialize something after the rest of the board has been configured